tests/integration/test_loki_push_api.py

```diff
@@ -72,8 +72,17 @@ async def test_rule_files_ingested_by_loki(ops_test):
     await ops_test.model.wait_for_idle(status="active", timeout=1000)
 
     # now, make sure rules are present
+    expected = [
+        ["HighPercentageError"],
+        ["HighThroughputLogStreams"],
+        ["HighThroughputLogStreams"],
+        ["HighThroughputLogStreams"],
+        ["http-credentials-leaked"],
+    ]
     response = await client.rules()
     assert (await client.rules()).items() > {}.items()
-    alert_group = next(iter(response.values()))[0]
-    assert "name" in alert_group
-    assert "rules" in alert_group
+    alerts = [
+        sorted([rule["alert"] for rule in group["rules"]])
+        for group in next(iter(response.values()))
```

Comment:
I don't remember the structure coming out in the response, but shouldn't it be
```suggestion
        for group in response.values()
```

Reply:
No values returns a list like object with one item that can't be subscripted. (Also this is how it was before)